probe/Cargo.toml

```diff
@@ -10,4 +10,13 @@ coresight = { path = "../coresight"}
 query_interface = "*"
 log = "0.4.6"
 bitfield = "*"
+bitflags = "*"
```

Comment:
Maybe we should start using specific versions now :wink:

Reply:
Will be fixed in the cargo-flash PR.